src/Imath/half.cpp

```diff
@@ -34,11 +33,23 @@ using namespace std;
 
 // clang-format off
 
-EXPORT_CONST const half::uif half::_toFloat[1 << 16] =
+#ifdef IMATH_ENABLE_HALF_LOOKUP_TABLES
+const imath_half_uif_t imath_half_to_float_table_data[1 << 16] =
 #include "toFloat.h"
-EXPORT_CONST const unsigned short half::_eLut[1 << 9] =
+
+const uint16_t imath_float_half_exp_table_data[1 << 9] =
 #include "eLut.h"
 
+extern "C" {
+EXPORT_CONST const imath_half_uif_t *imath_half_to_float_table = imath_half_to_float_table_data;
+EXPORT_CONST const uint16_t *imath_float_half_exp_table = imath_float_half_exp_table_data;
+} // extern "C"
+
+#endif
+
+// clang-format on
+
+//#ifdef IMATH_USE_ORIGINAL_HALF_IMPLEMENTATION
```

Comment:
Is there a better term than "ORIGINAL"? Does "original" == "lookuptable-based?" This may become confusing at some point in the future.